src/ebay-section-notice/section-notice.tsx

```diff
@@ -5,7 +5,7 @@ import React, {
     KeyboardEventHandler,
     MouseEvent,
     MouseEventHandler,
-    ReactElement, useState
+    ReactElement, useId, useState
```

Comment:
`useId` is only available on React 18, and we still have applications that runs in React 16, this will be a huge breaking change. For now we can do the same approach used on [EbayIcon](https://github.com/eBay/ebayui-core-react/blob/main/src/ebay-icon/icon.tsx#L27), where we use a state and a useEffect to set on mount. Reason we do on mount is to make sure the server and client rendering matches.